### PR TITLE
Update Lyft image & SMS 2FA support

### PR DESCRIPTION
**Pull request by Rfosu2k2**
Update Lyft image with smaller bytes and the correct brand guidelines found @:

https://s3.amazonaws.com/lyft-common/publicapi/production/swaggers/Lyft_Logo_2018.zip

Lyft also does support 2FA through SMS.

### CODE REVIEW

**Comment by Carlgo11** [comment]
Thanks for the pull request!

I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?

If no documentation is available or you can't find it, could you please add some screenshots of the 2FA setup process and/or login process?
__Remember to blur or crop out any personal information linked to you.__

You can attach the screenshots here in a comment, like this:
screenshot

File: _data/transport.yml [hunk 1]
```diff
@@ -110,6 +110,8 @@ websites:
 
   - name: Lyft
     url: https://www.lyft.com/
+    tfa:
+      - sms
     twitter: AskLyft
     facebook: lyft
```
Comment (kmpoppe) [inline _data/transport.yml]:
```suggestion
```

**Comment by Rfosu2k2** [comment]
lyft_1
lyft_2
lyft_3

**Comment by Carlgo11** [comment]
Thanks!

**Review by Carlgo11 — APPROVED** [review]

**Comment by Carlgo11** [comment]
I just looked through it myself. Looks like it's single factor actually.
The single factor just happens to be SMS instead of the normal password factor.

Sorry for the inconvenience!

**Comment by Rfosu2k2** [comment]
I guess it is only single factor since it doesn't require password.  Sorry for not clearly seeing this before making the request.  Closing....

**Pull request closed without merging**